configs/config.json.cluster.example

```diff
@@ -22,23 +22,6 @@
         "type": "HandleSoftBan"
       },
       {
-        "type": "CompleteTutorial",
```

Comment:
Not sure about the team ID since it's not required to pick a team, you can be level 30 without a team. And some people may not want to pick any team, I think